ui/src/tool-panel/location-track/splitting/location-track-split.tsx

```diff
@@ -71,6 +71,15 @@ export const LocationTrackSplit: React.FC<SplitProps> = ({
         split.descriptionBase !== '',
     );
 
+    React.useEffect(() => {
+        if (!nameErrors.length) {
+            setNameCommitted(true);
+        }
+        if (!descriptionErrors.length) {
+            setDescriptionCommitted(true);
+        }
+    });
```

Comment:
Sivuhuomiona paranneltu: Aiemmin kenttiin pystyi syöttämään vaikka mitä roskaa ja mahdolliset virheet tulivat esiin vasta kun kentästä hyppäsi pois. Nyt kentät commitoidaan heti kun niihin ilmestyy ensimmäistä kertaa validi arvo, joten kentän arvon muuttaminen validista tilasta epävalidiin tuo virheet heti esille.

Reply:
Ilmeisesti siis efektillä ei ole depsuarraytä ihan tarkoituksella?

Reply:
Hyvä huomio. Yleensä kirjoittelen kaikkialle depsuarrayt, mutta tämä on unohtunut kokonaan. Tämäkin toimii koska siitä ei ole mitään haittaa että tämä ajautuu joka renderillä, mutta voisin nyt vielä korjata tämän eksplisiittisemmäksi samalla kun teen muitakin UI-revikkakorjauksia

Reply:
...Iiiitse asiassa komento takaisin. Tässä tulee vastaan jotain melko kosmista Reactin kanssa, sillä minkä tahansa depsuarrayn tälle antaminen voi rerender-looppiin. Näin ei kuitenkaan tapahdu mikäli depsut on undefined. Näin tunti-pari ennen joululomalle lompsahtamista en ehdi paneutumaan tähän sen kummemmin, mutta jätän tähän koodikommentin.